jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/JavaCodeWhispererFileCrawler.kt

```diff
@@ -17,13 +17,10 @@ import com.intellij.psi.search.GlobalSearchScope
 import kotlinx.coroutines.yield
 import org.jetbrains.jps.model.java.JavaModuleSourceRootTypes
 
-// version1: Utilize PSI import elements to resolve imported files
 object JavaCodeWhispererFileCrawler : CodeWhispererFileCrawler() {
     override val fileExtension: String = "java"
-    override val testFilenamePattern: Regex = """(?:Test([^/\\]+)\.java|([^/\\]+)Test\.java)$""".toRegex()
     override val dialects: Set<String> = setOf("java")
-
-    override fun guessSourceFileName(tstFileName: String): String = tstFileName.substring(0, tstFileName.length - "Test.java".length) + ".java"
```

Comment:
moved to parent class https://github.com/aws/aws-toolkit-jetbrains/pull/3774/files#diff-faff4822410665507967aaa15bfe64abab03286e73a1bdc9c9d5046408de467bR139